src/audio/mixin_mixout/Kconfig

```diff
@@ -3,6 +3,7 @@
 config COMP_MIXIN_MIXOUT
 	tristate "Mixin_mixout component"
 	depends on IPC_MAJOR_4
+	default m if LIBRARY_DEFAULT_MODULAR
```

Comment:
please select a test module not used in typical topologies

Reply:
> please select a test module...

`src/audio/mixin_mixout/Kconfig` is not a "test" file, this line is not changing a "test" configuration, it's changing the default, production configuration (which of course happens to be tested).

I have no idea whether this change is correct or not but I want to be sure there is no misunderstanding: this is not just a "test" change.

Reply:
> this is not just a "test" change.

On MTL this is a "test" change and there is no plan for such production configuration of mixin (at least for now), that's why I asked for another "test" module (not used in production topologies).

On the other hand, this may be discussed for future platforms to be used as production configuration.